src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace SMG\ManagerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\Controller\Annotations;
+use SMG\UserBundle\Entity\User;
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\Controller\FOSRestController;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+
+class ManagersController extends FOSRestController
+{
+    use Traits\TokenFromHeaderTrait;
+    use Traits\HandleUserTrait;
+
+    const TOKEN_DIGITS = 6;
+
+    /**
+     * @Annotations\Post("/user/add")
+     *
+     * @ParamConverter("user", converter="fos_rest.request_body")
+     */
+    public function createUserAction(User $user)
+    {
+        $this->checkUserClientType('backend');
+        $manager = $this->get('fos_user.user_manager');
```

Comment:
except setEnabled and setLocket,  all the part below can certainly be put in the trait